Sources/MessagingInApp/Gist/Managers/ModalViewManager.swift

```diff
@@ -10,8 +11,10 @@ class ModalViewManager {
     var window: UIWindow?
     var viewController: GistModalViewController!
     var position: MessagePosition
-    var isShowingMessage: Bool {
-        window != nil
+    var isShowingMessage: Bool = false // true if the modal is visible to the user. Meaning, all operations are complete including animations.
```

Comment:
One part of making the tests more reliable is changing `isShowingMessage` to only be `true` if *all* of the animations have been completed. 

In an app, we do not consider a message as shown until all View animations are completed. Our tests should match that and now they do.